ui/src/app/shared/status/status.icon.html

```diff
@@ -6,5 +6,5 @@
     <i class="remove red icon" *ngSwitchCase="'Fail'"></i>
     <i class="ban grey icon" *ngSwitchCase="'Disabled'"></i>
     <i class="ban grey icon" *ngSwitchCase="'Skipped'"></i>
-    <i class="wait orange icon" *ngSwitchCase="'Waiting'"></i>
+    <i class="wait blue icon" *ngSwitchCase="'Waiting'"></i>
```

Comment:
Must override .blue.icon ( semantic icon color) to use $cds_color_teal  => style.scss
Missing TU impact: status.spec.ts and workflow.spec.ts